src/main/java/seedu/address/MainApp.java

```diff
@@ -73,9 +67,8 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
         Optional<ReadOnlyQuickCache> addressBookOptional;
         ReadOnlyQuickCache initialData;
 
-        initialData = SampleDataUtil.getSampleQuickCache(); // can delete after storage is implemented
+        // initialData = SampleDataUtil.getSampleQuickCache(); // can delete after storage is implemented
```

Comment:
Why is this deletable though? I thought this was the initial data to be used to populate a brand new instance of QuickCache

Reply:
I think this method is to initialise sample question. But will not be implemented last after v1.2a

Reply:
okay great